resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStep.scala

```diff
@@ -102,7 +102,7 @@ private[spark] class BasicExecutorFeatureStep(
     // hostname must be no longer than 63 characters, so take the last 63 characters of the pod
     // name as the hostname.  This preserves uniqueness since the end of name contains
     // executorId
-    val hostname = name.substring(Math.max(0, name.length - 63))
+    val hostname = name.substring(Math.max(0, name.length - KUBERNETES_LABEL_MAX_LENGTH))
```

Comment:
After this change, line 102 looks like a dangling comment. Could you revise it accordingly?

Reply:
OK